README.md

```diff
@@ -1,13 +1,7 @@
-# Project Name
-
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+# Guess who
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
-
 ## View it live
 
-Have you deployed your project somewhere? Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://affectionate-colden-2160ca.netlify.app/
```

Comment:
Your project shines like a diamond 💎  but nobody will know about it if you will not have a great invitation here. ReadMe file is the cover of your code. It is the first impression. Polish it as hard as you polished your code 🎨